salt/utils/vault.py

```diff
@@ -111,8 +111,8 @@ def _use_local_config():
 
     if 'vault' in __opts__ and __opts__.get('__role', 'minion') == 'master':
         return _use_local_config()
-    elif '_ssh_version' in __opts__:
-        return _use_local_config()
+    elif any((__opts__['local'], __opts__['file_client'] == local, __opts__['master_type'] == 'disable')):
+        _use_local_config()
```

Comment:
Did you mean to delete the `return`?

Reply:
yes it does, and i have no idea where it went, cause i only changed the _ssh_version line